Library/Formula/repo.rb

```diff
@@ -3,9 +3,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage 'http://source.android.com/source/git-repo.html'
-  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.13/repo'
+  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo'
   version '1.21'
```

Comment:
I see. Maybe it's worth adding `revision 1` to the formula so people who think they're currently using version `1.21` actually are?